docs/book/src/tasks/certs/experimental-ca-rotation.md

```diff
@@ -0,0 +1,112 @@
+## Experimental CA rotation procedure
+
+<aside class="note warning">
+
+<h1>Cluster-api CA rotation support</h1>
+
+CA rotation is not supported (nor automated) by cluster-api, please use this procedure at your own risk!
+Rotating CA is a tough Kubernetes topic on its own, it's even harder when everything is wrapped around cluster-api automation and immutability concepts.
+
+An issue is currently tracking discussions around this feature request : [Automatic CA rotation in CAPI #7721](https://github.com/kubernetes-sigs/cluster-api/issues/7721). Reading the issue is strongly advised to really be sure of what you are going to do.
+
+Cluster-api ultimate goal is to support such feature. But at the moment, the best (and cleaner) strategy is still to destroy and recreate your managed cluster when the CA is expired.
+
+</aside>
+
+## CA manual rotation step-by-step guide
+
+### Rotating `cluster-ca`, `cluster-proxy` & `cluster-etcd` CA Secrets
+
+<aside class="note">
+
+<h1>Keeping things shorter</h1>
+
+During steap we're going to talk about CA1 or 2, pubCA and CAkey. Let's be clear about it:
+
+* CA "1" = old CA that needs to be replaced
+* CA "2" = new CA that will replace the old one and signs all new kubernetes certificates
+* pubCA "1+2" = means that you should concatenate CA cert1 and CA cert2 (in this order)
+* CAkey2 = this is the private key for the new CA
+
+</aside>
+
+**PREFLIGHT CA ROTATION OPERATIONS :**
+
+Managed cluster needs to recognize the incoming new CA before switching all control-plane nodes all at once.
+By injecting the new CAcert as bundle in all kubernetes components and rolling everything, we ensure that kubernetes api & kubelet will still work immediately after the control-plane cert rotation.
+
+- [ ] a working & stable managed cluster should have 4 secrets provisionned with "old" CAs
+- [ ] backup/copy existing Secret as `[cluster name]-**-backup` to keep old state of the cluster & allow recovery
+- [ ] update CA Secret with pubCA 1+2 and CAkey 1
+- [ ] update PROXY Secret with pubPROXY 1+2 and PROXYkey 1
```

Comment:
We should add all the terms used e.g. `pubETCD`, `pubPROXY` to the glossary above.